src/Skills.jsx

```diff
@@ -161,6 +161,9 @@ class Skills extends Component {
          },
        });
      })
+     .then(() => {
+       browserHistory.replace('/skills/');
```

Comment:
That's a violation of the model though. It better that the user back up and get a 404, confirming that the page is indeed gone, than her not being able to find history of where she has been.